sys/boot/geli/geliboot.h

```diff
@@ -27,35 +27,11 @@
  * $FreeBSD$
  */
 
-#include <sys/endian.h>
```

Comment:
the amount of direct includes here, like md5 etc, made this file too hefty to include in said i386/loader/main.c
So I broke it up into geliboot.h (prototypes etc), and geliboot_internal.h (stuff for geliboot itself)